sql/moz-fx-data-shared-prod/pocket/spoc_tile_ids/metadata.yaml

```diff
@@ -3,3 +3,7 @@ description: |-
   Lookup table of the currently registered tile IDs for Pocket sponsored content
 owners:
   - jklukas@mozilla.com
+workgroup_access:
```

Comment:
Apologies for not looking at the CI more closely. It appears the lint failure is as follows:

```
./sql/moz-fx-data-shared-prod/pocket/spoc_tile_ids/metadata.yaml
  7:1       error    wrong indentation: expected 2 but found 0  (indentation)
  9:3       error    wrong indentation: expected 4 but found 2  (indentation)

./sql/moz-fx-data-shared-prod/pocket_derived/spoc_tile_ids_v1/metadata.yaml
  13:1      error    wrong indentation: expected 2 but found 0  (indentation)
  15:3      error    wrong indentation: expected 4 but found 2  (indentation)
```

So the indentation needs to be:
```yaml
workgroup_access:
  - role: roles/bigquery.dataViewer
    members:
      - workgroup:pocket/prefect
```